### ISSUE TITLE
[Feature Request] Synthetics Monitor Schedule

### ISSUE THREAD

**Issue by RaphSimard** [elastic/synthetics#895]
Hi,
I will start by mentioning that Synthetic Monitors are awesome and that they completely transformed how we monitor the availability of our services. Truly a game changer.
Although, please add the possibility to add a schedule for a synthetic monitor. 
For example, we have many ICMP monitors for our servers and those servers reboot monthly. 
Every time they do, we receive alerts in Elastic from the monitor. 
I would like to be able to configure the monitors so that they don't run on specific hours and days of the month.
Creating a maintenance window does not works since it blocks all outgoing alerts which is not what I want to do.

Thank you very much !

**Comment by MakoWish** [comment]
Have you looked into Maintenance Windows? 

`https://cluster_url.contoso.com/app/management/insightsAndAlerting/maintenanceWindows/create`

You could create a Maintenance Windows and filter specifically for the Synthetics monitors you do not want alerted on during your monthly maintenance schedule. The monitors will still run, and the downtime will still be recorded, but you will not receive alerts during that time.

**Comment by RaphSimard** [comment]
Hi @MakoWish, 
I was not aware of the filter options for the maintenance windows.
This is exactly what I was looking for.
Thank you very much.

**Comment by RaphSimard** [comment]
@MakoWish : I am not sure which field I need to filter on so that it targets a specific Synthetic Monitor.
I can see fields like kibana.alert.X but I think I should be filtering for something like monitor.id.
Do you have an idea ?

**Comment by MakoWish** [comment]
Try `kibana.alert.rule.name` to filter for your specific rule you want muted during the maintenance window.

**Comment by RaphSimard** [comment]
I do not think this would work since it would mute all of my Synthetics monitors...
This goes back to what I was mentioning in my first post.
There is only 1 Kibana rule for my 200+ Synthetics monitors.

![image](https://github.com/elastic/synthetics/assets/54416260/e966f4f8-e64b-43ee-be08-78c122214b69)

**Comment by MakoWish** [comment]
You didn't mention that it was a single monitoring rule before, so that definitely does make things more difficult. What you are asking for would have to look at the {{context.hits}} when there are matches and not send the alert for any matches. That may be a difficult ask, but I will leave that up to the DEVs to chime in. 

I would suggest breaking your one monitoring rule out to the different groups of systems you want to monitor separately. We have different monitoring rules for not only our different teams, but different levels or types of systems. For instance, our AppDev team alone has Sandbox, Development, Staging, and Production. Each of those groups of systems have their own monitoring rule. We have close to a thousand total monitors with several dozen different monitoring rules. This way, we can have maintenance windows for each of those different groups of systems. 

If you want to keep this ER open, by all means do, but it may take a long time (years?) before anything like this becomes available; if ever. Breaking that one monitoring rule out into groups of systems will be your best bet in the short term.

**Comment by RaphSimard** [comment]
The single monitoring rule what created automatically so I though it was the same for everyone. Sorry for the confusion.
Thank your input on this. It is greatly appreciated. Especially since you are the only one that took the time to leave a comment.

I really hope they add this feature because I find it to be pretty important. I imagine that everyone that is using synthetics monitoring would want this to be added.

**Comment by MakoWish** [comment]
The Maintenance Window feature was one that we voted for and applauded when released, but it seems to work well for us since we have separate monitoring rules for each different group of devices.
